Digital_guj_scraper.py

```diff
@@ -1,56 +1,23 @@
-import os
-import requests 
+import requests
+import json
 from bs4 import BeautifulSoup
+import os
+from os import path
+import pickle
```

Comment:
remove the modules which are not use and also update requirements.txt if any new module is added.